packages/mobile/src/utils/formatting.ts

```diff
@@ -20,6 +21,19 @@ export const getMoneyDisplayValue = (
   return includeSymbol ? symbol + formattedValue : formattedValue
 }
 
+export const getLocalCurrencyDisplayValue = (
```

Comment:
How does this differ than `getMoneyDisplayValue` above? Can the two be merged?

Reply:
`getMoneyDisplayValue` uses `currency`(cUSD, cGOLD), `getLocalCurrencyDisplayValue` uses `localCurrency`(EUR, etc.). It looks weird for me as well, but our current currency logic is separated between these two, so I decided to keep it this way.

Reply:
Yes we'll probably merge them once we tackle the local currency refactor.